jobs/async-upload/job/config.py

```diff
@@ -149,11 +154,13 @@ def _load_oci_credentials(
         raise ValueError("Invalid docker config file")
 
     # Load the credentials from the docker config, the URI passed in via config is used as a key to find the correct credentials
-    auth = docker_config["auths"][store["uri"]]["auth"]
-    username, password = base64.b64decode(auth).decode("utf-8").split(":")
-    store["username"] = username
-    store["password"] = password
-    store["email"] = docker_config["auths"][store["uri"]]["email"]
+    registry = store["oci"]["registry"]
+    auth = docker_config["auths"][registry]["auth"]
+    # TODO: This might not be the correct way to parse this
+    username, password = auth.split(":")
```

Comment:
Ive seen the credentials be uplaoded as base-64 encoded, but in other places ive seen it passed in as plain-text. I'm not sure which is the "correct" way, but for now i will assume plain-text